packages/manager/modules/vrack/src/vrack-order.js

```diff
@@ -8,7 +8,7 @@ export const VRACK_ORDER_URLS = {
   FI:
     "https://www.ovh-hosting.fi/order/express/#/new/express/resume?products=~(~(planCode~'vrack~quantity~1~productId~'vrack))",
   FR:
-    "https://www.ovh.com/fr/order/express/#/new/express/resume?products=~(~(planCode~'vrack~quantity~1~productId~'vrack))",
+    "https://www.ovh.com/fr22222/order/express/#/new/express/resume?products=~(~(planCode~'vrack~quantity~1~productId~'vrack))",
```

Comment:
**question:** This looks like an unwanted change. are you sure about this?

Reply:
Yup. Will remove it Thanks Anoop.